components/filter-components/MapFilters.vue

```diff
@@ -13,7 +13,8 @@
              style="width:90%">
 
           <!-- Header -->
-          <div class="flex w-full flex-col"
+          <div class="flex w-full flex-col sticky top-0 z-10
```

Comment:
Can't remember how you do this think it may be something like:

md:sticky md:top-0 md:z-10

But ideally, we will only make the title sticky on non-mobile/small browsers. Think the title may take up a bit too much real estate on mobile devices, and you can't see the filters that well. So probably worth applying on larger browsers only. Hope that makes sense.

Reply:
Sorry, think my original code comment was hidden. The changes don't seem to work for me.

But the following works:

 <div class="flex w-full flex-col bg-white md:sticky md:top-0 md:z-10"
               ref="FiltersHeader">
            <div class="flex flex-col w-full h-full"
                 style="backdrop-filter: blur(1.5px);">

Can you please try replacing what you have with this. See if it works, and if it does, I'll merge. And thanks for looking at this.